doc/source/installation.rst

```diff
@@ -1,130 +1,126 @@
 
-    If you are installing on Windows, you must have
-    Microsoft Visual C++ 14.0 installed. If your installation
-    fails with the error message:
+    - If you encounter errors on **Windows** related to **Microsoft Visual C++ 14.0**, install the required **Build Tools for Visual Studio** from: `Microsoft Visual Studio Downloads <https://visualstudio.microsoft.com/downloads/>`_. 
+    - If you encounter any other issues following these instructions, seek help on `GitHub Discussions (Installation) <https://github.com/MDAnalysis/mdanalysis/discussions/categories/installation>`_.    
 
-        error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio": https://visualstudio.microsoft.com/downloads/
-
-    Try installing Build Tools for Visual Studio from
-    https://visualstudio.microsoft.com/downloads/ (scroll
-    down to the Tools for Visual Studio section).
-
-
-If you encounter any issues following these instructions, please
-ask for help on `GitHub Discussions (Installation)`_.
-
-.. _`GitHub Discussions (Installation)`: https://github.com/MDAnalysis/mdanalysis/discussions/categories/installation
+.. warning::
+    `Conda <https://docs.conda.io/projects/conda/en/latest/>`_ and `pip <https://pip.pypa.io/en/latest/index.html>`_ installations **do not include** external programs such as `HOLE <https://www.holeprogram.org//>`_.
 
-conda
+Conda installation
 =====
-
-If you use ``conda`` to manage your Python environment, we highly recommend creating a new environment for MDAnalysis.
-This will ensure that you have a clean installation of MDAnalysis and its dependencies, and will not interfere with other packages you may have installed.
-We further recommend that you install and use ``mamba``, a faster drop-in replacement for ``conda``.
+ 
+For most users, `conda <https://docs.conda.io/projects/conda/en/latest/>`_ is the easiest way to install MDAnalysis because it manages dependencies effectively. 
+We highly recommend creating a new environment for MDAnalysis. We further recommend that you install and use ``mamba``, a faster drop-in replacement for ``conda``.
 
 .. code-block:: bash
 
     conda create --name mdanalysis
     conda activate mdanalysis
     conda install -c conda-forge mamba
 
-To install the latest stable version of MDAnalysis via ``conda``, use the following command. This installs all dependencies needed for full analysis functionality (excluding external programs such as `HOLE`_):
+To install the latest stable version of MDAnalysis via ``conda``, use the following command. This installs all dependencies needed for full analysis functionality.
 
 .. code-block:: bash
 
     mamba install -c conda-forge mdanalysis
 
-To upgrade:
+To upgrade use:
 
 .. code-block:: bash
 
     mamba update mdanalysis
 
-To install the tests:
+To install the `test suite <https://userguide.mdanalysis.org/stable/datasets.html#mdanalysistests>`_ use:
```

Comment:
It might be easier to use a `:ref:` here like above, which should automatically resolve to the link!